esmvalcore/cmor/_fixes/native6/ipsl_cm6.py

```diff
@@ -0,0 +1,114 @@
+"""Fixes for IPSLCM6 TS output format."""
+import logging
+import subprocess
+import time
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+# The key used in extra_facets file for providing the
+# variable name (in NetCDF file) that match the CMOR variable name
+VARNAME_KEY = "ipsl_varname"
+
+
+class AllVars(Fix):
+    """Fixes for all IPSLCM variables."""
+
+    def fix_file(self, filepath, output_dir):
+        """Select IPSLCM variable in filepath, by calling CDO, if relevant.
+
+        This is done only if input file is a multi-variable one. This
+        is diagnosed by searching in the input filepathame for the
+        extra_facet value for key 'group'.
+
+        In such cases, it is worth to use an external tool for
+        filtering, at least until Iris loads fast (which is not the case
+        up to, and including, V3.0.2)
+
+        However, we take care of ESMValTool policy re. dependencies licence
+
+        """
+        if "_" + self.extra_facets.get("group",
+                                       "non-sense") + ".nc" not in filepath:
+            # No need to filter the file
+            logger.debug("In ipsl-cm6.py : not filtering for %s", filepath)
```

Comment:
```suggestion
            logger.debug("Not filtering for %s", filepath)
```
there is no need to mention the filename in log messages, this is automatically recorded by the logging system in main_log_debug.txt

Reply:
OK. Done